site/assets/scss/_toc.scss

```
@@ -26,7 +26,7 @@
 
     a {
       display: block;
-      padding: .125rem 0 .125rem .75rem;
+      padding: .313rem 0 .25rem .313rem;
```

If we modify values from Bootstrap, we should add the "Boosted mod" comment.

```suggestion
      padding: .313rem 0 .25rem .313rem; // Boosted mod
```

---

`.313rem` seems weird to me as a value as it's `5.008px`

---

Sorry I think I was a bit tired last evening 😓